ContentPatcher/docs/i18n/zh/author-guide/action-editmap.md

```diff
@@ -0,0 +1,383 @@
+﻿← [模组作者指南](../author-guide.md)
+
+一个含有 **`"Action": "EditMap"`** 的补丁会更改游戏已加载的地图的一部分。任意数量的内容包都可以编辑同一素材。你可以用补丁向下和向右延伸地图（Content Patcher将扩展地图以适应新地图）。
+
+**🌐 其他语言： [en (English)](../../../author-guide/action-editmap.md)。**
+
+## 目录
+* [介绍](#introduction)
+  * [什么是地图？](#what-is-a-map)
+* [用法](#usage)
+  * [概述](#overview)
+  * [公共字段](#common-fields)
+  * [地图叠加](#overlay-a-map)
+  * [编辑地图属性](#edit-map-properties)
+  * [编辑地图图块](#edit-map-tiles)
+* [已知限制](#known-limitations)
+* [参见](#see-also)
+
+## 介绍<a name="usage"></a>
+### 什么是地图？<a name="what-is-a-map"></a>
+一个地图素材描述游戏内某个区域的的地形（水，悬崖，地面），地形特征（灌木），建筑，路径，和触发点。当屏幕在你到达某个区域的边缘或进入建筑物时变黑时，你正在从一个地图移动到另一个地图。
+
+**维基上的[模组:地图](https://zh.stardewvalleywiki.com/模组:地图)**有更详细的介绍地图的入门和进阶概念。
+
+## 用法<a name="usage"></a>
+### 概述<a name="overview"></a>
+每一个`EditMap`补丁可以对某一地图进行三种类型的更改：叠加地图、更改地图属性或更改地图图块。
+
+这三种类更改型的效果大相径庭，所以分段描述，但是它们可以同时出现在一个补丁。
+一个补丁补丁中的字段以此顺序生效：`FromFile`，`MapTiles`，`MapProperties`，`AddWarps`，和`TextOperations`.
+
+### 公共字段<a name="common-fields"></a>
+一个`EditImage`补丁是`Changes`下含有此字段的模型 (见例子)。所有更改类型都需要这些字段。
+
+<dl>
+<dt>必填字段：</dt>
+<dd>
+
+字段       | 用途
+--------- | -------
+`Action`  | 要进行的更改类型。此操作类型设置为`EditMap`。
+`Target`  | 需编辑的[游戏素材名](../author-guide.md#what-is-an-asset)（或多个由逗号分隔的素材名），比如`Maps/Town`。该字段支持[令牌](../author-guide.md#tokens)，不区分大小写。
+
+</dd>
+<dt>可选字段：</dt>
+<dd>
+
+字段       | 用途
+--------- | -------
+`When`      | _(可选)_ 使此补丁只有在指定[条件](../author-guide.md#conditions)下生效.
+`LogName`   | _(可选)_ 此补丁在日志里显示的名字，有助于理解报错。默认为类似`EditImage Maps/Town`的名字。
+`Update`    | _(可选)_ 此补丁字条的更新频率，详见[update rate](../author-guide.md#update-rate)。
+`LocalTokens` | _(可选)_ 一组仅在此补丁中生效的[局部令牌](../author-guide/tokens.md#local-tokens)。
+
+
+</dd>
+<dt>进阶字段：</dt>
+<dd>
+
+<table>
+  <tr>
+    <td>字段</td>
+    <td>用途</td>
+  </tr>
+  <tr>
+  <td><code>Priority</code></td>
+  <td>
+
+ _（可选）_ 当多个补丁编辑同一数据素材时，此字段控制它们应用的顺序。可用的值有`Early`（更早），`Default`（默认），还有`Late`（更晚）。默认值为`Default`。
+
+补丁（包括所有模组）按以下顺序生效：
+
+1. 优先级从早到晚；
+2. 按照模组加载顺序（基于依赖关系）；
```

Comment:
> 2. 按照模组加载顺序（基于依赖关系）；

Note that dependencies are an example of what can affect load order, but the order also depends on factors like filesystem path. That's why the original text has the _e.g._ in "_(e.g. based on dependencies)_".